pkg/resource/any.go

```diff
@@ -18,12 +18,12 @@ type Any struct {
 
 type anySpec struct {
 	value interface{}
-	doc   yaml.Node
 	yaml  []byte
 }
 
-func (s anySpec) MarshalYAML() (interface{}, error) {
-	return s.doc.Content[0], nil
+// GetBytes implements RawYAML interface.
+func (s anySpec) GetBytes() ([]byte, error) {
```

Comment:
Should it be called better `MarshalYAMLBytes` or close to that?

Reply:
yeah, I like that variant. Should be less chances to break the code for someone if they use that function name already.

Reply:
yeah, I like that variant. Should be less chances to break the code for someone if they use that function name already.